.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Ensure the title of this issue starts with the component name followed by a colon, e.g.
+> ncm-example: demonstrate what titles should look like
+
+Describe the problem here.
```

Comment:
Describe the problem here, remembering to include:
- what version of the component you are using (rpm -q $component)
- what OS version you are using (/etc/redhat-release or similar)
- steps to reproduce
- actual result versus expected result

Or something similar? (I guess not all issues are bug reports so perhaps this is a bit too specific?)